controls/views.py

```diff
@@ -206,6 +206,29 @@ def controls_updated(request, system_id):
         # User does not have permission to this system
         raise Http404
 
+def rename_element(request,element_id):
+    """Update the component's name
+    Args:
+        request ([HttpRequest]): The network request
+    component_id ([int|str]): The id of the component
+    Returns:
+        [JsonResponse]: Either a ok status or an error 
+    """
+    try:
+        new_name = request.POST.get("name", "").strip() or None
+        element = get_object_or_404(Element, id=element_id)
```

Comment:
not controls_element(Element), edit the siteapp_portfolio table(Portfolio). Which includes description and title.

Reply:
Editing component and editing portfolio are two different tasks. Both are needed.